Assets/Scripts/Controllers/Sprites/FurnitureSpriteController.cs

```diff
@@ -324,6 +335,7 @@ private string GetSuffixForDiagonalNeighbour(string suffix, string coord1, strin
     {
         if (suffix.Contains(coord1) && suffix.Contains(coord2))
         {
+            // FIXME: Doing ToLower here sucks!
```

Comment:
Coordinates can only be N,E,S,W so lookup table if it really is a perf concern could also act as a validation measure.